### PR TITLE
set deneb fork on testnets

### PR DESCRIPTION
**Pull request by realbigsean**
## Issue Addressed

Setting some deneb forks!

Resolves https://github.com/sigp/lighthouse/issues/5055

I downloaded the config files these PRs/sources:

Holesky: https://github.com/eth-clients/holesky/pull/94, https://github.com/eth-clients/holesky/pull/95
Sepolia: https://github.com/eth-clients/sepolia/pull/61
Chiado: https://github.com/gnosischain/configs/blob/main/chiado/config.yaml

### CODE REVIEW

File: common/eth2_network_config/built_in_network_configs/chiado/config.yaml [hunk 1]
```diff
@@ -150,7 +43,95 @@ BELLATRIX_FORK_EPOCH: 180 # Mon Oct 10 2022 14:00:00 GMT+0000
...
+CHURN_LIMIT_QUOTIENT: 4096
+# [New in Deneb:EIP7514] 2*
+MAX_PER_EPOCH_ACTIVATION_CHURN_LIMIT: 2
+
+# Fork choice
+# ---------------------------------------------------------------
+# 40%
+PROPOSER_SCORE_BOOST: 40
+# 20%
+REORG_HEAD_WEIGHT_THRESHOLD: 20
```
Comment (pawanjay176) [inline common/eth2_network_config/built_in_network_configs/chiado/config.yaml:85]: Just curious, we don't define this in the lighthouse ChainSpec. Do we need to?!

**Review by pawanjay176 — APPROVED** [review]
LGTM.

File: common/eth2_network_config/built_in_network_configs/chiado/config.yaml (hunk 1 above)
Comment (dapplion) [reply common/eth2_network_config/built_in_network_configs/chiado/config.yaml:85]: It's defined on the mainnet one https://github.com/ethereum/consensus-specs/blob/3727a757450feb6110e2d4e403604572a478343d/configs/mainnet.yaml#L98-L100 but probably not used until 
- https://github.com/sigp/lighthouse/pull/4887

**Review by dapplion — APPROVED** [review]
LGTM

**Pull request merged**